Enable rendering Symfony Forms fields in `FieldGroup` (#1283)
lmc-eu/spirit-design-system: packages/web-twig/src/Resources/components/FieldGroup/README.md

⚠️ **The FieldGroup component does not provide all necessary semantics and any styling to its child fields. It is up
to the developer to configure the child fields correctly.**

⚠️ Remember the FieldGroup component required to be used with content.

> Otherwise, it is useless. So if used dynamically, make sure to have removed the FieldGroup correctly if there is no content.

Author: ```suggestion
Otherwise, it is useless. If it is used dynamically, make sure the component without content has been removed correctly.
```

But I'm still struggling with English grammar, so maybe @adamkudrna will have better suggestion.

Reviewer: Fixed.
